src/Components/BehaviorsModal/Internal/Widgets/GaugeWidget/GaugeWidget.tsx

```diff
@@ -32,12 +34,11 @@ const GaugeWidget: React.FC<IProp> = ({ widget }) => {
             if (!value) {
                 value = 0;
             }
-            value = Math.floor(value);
         }
     } catch {
         value = 0;
     }
-
+    formatedValue = formatNumber(value);
```

Comment:
super nit. can we just instantiate the variable here since it's not set before this?
`const formattedValue`
Another super nit, formatted has 2 't's. 🙂